db/migrate/20170913131914_create_active_admin_comments.rb

```diff
@@ -1,17 +0,0 @@
-class CreateActiveAdminComments < ActiveRecord::Migration::Current
```

Comment:
Why is this delete here? maybe it's from another PR?

Reply:
oh yes! it's in the other PR, i should merge